postgres-connector/resources/test/docker/vanilla-schema-image/create_database.sql

```diff
@@ -1,1284 +0,0 @@
---
```

Comment:
do we need any kind of license spec/copyright in this file?

Reply:
I actually wrote these from scratch

Reply:
If so the usual egeria copyright/license would apply

Reply:
This file looks like the dvd rentals database?

Reply:
That was the data I used to test while I was coding , but this PR deletes that file and replaces it with two SQL scripts that I wrote from scratch ?

Reply:
Duh! Apologies - was looking at deleted file!